samples: wifi: Update command output and help (#10346)
nrfconnect/sdk-nrf: samples/wifi/shell/README.rst

       | No argument - Prints current configuration
       | on - Turns on power save feature
       | off - Turns off power save feature

>    * - ps_mode
>      - | Configure Wi-Fi power save mode legacy/wmm
>        | 0 - Legacy
>        | 1 - WMM

Reviewer: ```suggestion
   * - ps_mode
     - | Configure Wi-Fi power save mode
       | 0 - Legacy
       | 1 - WMM
```

       | No argument - Prints current configuration
       | on - Turns on power save feature
       | off - Turns off power save feature
   * - ps_mode

>      - Configure Wi-Fi power save mode legacy/wmm

Reviewer 2: Can't we list the values for legacy and wmm? Also wmm should be WMM.
e.g.
- | Configure Wi-Fi power save mode
  | 0 - Legacy
  | 1 - WMM

       | <dialog_token> <flow_id>
       |
       | teardown_all - Teardown all TWT flows

>    * - reg_domian

Reviewer 2: reg_domain*

>      - | Set or Get Wi-Fi regulatory domain

Reviewer 2: get*

       | <dialog_token> <flow_id>
       |
       | teardown_all - Teardown all TWT flows
   * - reg_domain
     - | Set or get Wi-Fi regulatory domain
       |
       | Usage: wifi reg_domain [ISO/IEC 3166-1 alpha2] [-f]
       |
       | -f: Force to use this regulatory hint over any other regulatory hints.

>        | Note: This may cause regulatory compliance issues, use it at your
>        | own risk.

Reviewer: ```suggestion
       | (Note that this may cause regulatory compliance issues.)
```
